.github/workflows/release.yml

```diff
@@ -131,9 +137,10 @@ jobs:
         env:
           RELEASE_NOTES: ${{ fromJSON(steps.get-release.outputs.result).body }}
           RELEASE_URL: ${{ fromJSON(steps.get-release.outputs.result).url }}
+          COMMIT_DETAILS: ${{ needs.version-tag.outputs.commit_details }}
```

Comment:
It's better to provide a more detailed message in the comment for context. Consider mentioning why the PUBLISH_DOCS variable is set to true when a relevant commit message is found.